src/app/view/components/_common/action-menu-button/action-menu-button.component.spec.ts

```diff
@@ -220,12 +220,14 @@ describe('ActionMenuButtonComponent', () => {
             owner: '123',
             system: false
         };
-
         comp.actions = fakeActions;
         comp.item = mockContentType;
         fixture.detectChanges();
+        const actionButtonMenu = de.query(By.css('.dot-menu__button'));
+        actionButtonMenu.triggerEventHandler('click', {});
+        fixture.detectChanges();
 
-        const menuItemsLink = de.queryAll(By.css('.ui-menu-list a'));
+        const menuItemsLink = de.queryAll(By.css('.dot-menu-item__link'));
         menuItemsLink[1].nativeElement.click();
 
         expect(fakeCommand).toHaveBeenCalledTimes(1);
```

Comment:
I think we need one more test to check the model and icon attribute in dot-menu

Reply:
currently dot-menu do not support icons.